.gitignore

```diff
@@ -100,4 +100,8 @@ ENV/
 .build
 
 # pyenv
-.python-version
\ No newline at end of file
+.python-version
+
+# Design System templates
+frontstage/templates/components
```

Comment:
These are now ignored because I don't think the DS templates should be checked into the RASRM repo because they aren't really part of the frontstage app codebase and should be pulled in each time